src/Makefile.vc6

```diff
@@ -23,7 +23,7 @@ USE_DEF_FILE     = 1
 # '%INCLUDE%' variable instead of setting it here.
 #
 USE_IP2LOCATION  = 1
-IP2LOCATION_ROOT = C:\Users\Username\projects\IP2Location-C-Library
+IP2LOCATION_ROOT = $(MAKEDIR)\..\IP2Location-C-Library
```

Comment:
`$(MAKEDIR)`? Some kind of Nmake built-in? Never heard of, but better than hard-coded paths.

Reply:
Yes. I found it deeply buried on [MSDN](https://msdn.microsoft.com/en-us/library/7cafx990.aspx). It resolves to the current directory.